### PR TITLE
Fwsp1 43

### PR DESCRIPTION
**Pull request by googletorp**
No tests on this PR.

Made a custom block for the menu block, to add template suggestion, was the only way I could make a custom template for the menu. I made the template based on starting menu level, seemed like the best option available to have something remotely reusable.

### CODE REVIEW

File: web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js [hunk 1]
```diff
@@ -5,5 +5,18 @@
         $(".menu--main").toggleClass('active');
       })
     }
+  };
+
+  Drupal.behaviors.kkBasisNavigationPage = {
+    attach: (context, settings) => {
+      console.log('test');
+      console.log($(context).find('.menu--navigation-page .toggler'));
```
Comment (naxoc) [inline web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js:13]: A couple of `console.log`s

File: web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js [hunk 2]
```diff
@@ -5,5 +5,18 @@
...
+      $(context).find('.menu--navigation-page .toggler').click(function() {
+        let currentText = $(this).text();
```
Comment (naxoc) [inline web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js:15]: Should be `const`?

File: web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+.page-node-type-navigation {
+  // Custom implementation of secondary_background_color since it doesn't work
+  // on body element.
+  @each $key, $color_scheme in $color_schemes {
+    &.color-scheme-#{$key} {
+      background-color: map-get($color_scheme, 'light');
+    }
+  }
+
+  h1 {
```
Comment (naxoc) [inline web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss:11]: Is that not a little too generic?

Comment (googletorp) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss:11]: There should only one h1 on the page

Comment (googletorp) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss:11]: the class above is a body class, targeting all nav pages

File: web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js (hunk 2 above)
Comment (googletorp) [reply web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js]: True

File: web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js (hunk 1 above)
Comment (googletorp) [reply web/themes/custom/kk_basis/src_assets/js/kk_basis.main.js]: OOpps

File: web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss (hunk 3 above)
Comment (naxoc) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss:11]: But why would there only be one h1? With the `<article>` tag in html5 more than one is encouraged?

Comment (googletorp) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss:11]: I'm old school SEO trained - never more than one h1 per page - do you want to use multiple h1 tags?

Comment (naxoc) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss:11]: I think it is hard to assume that there will only ever be one. But in an `article` tag, the title should be a `h1`. And the `article` tag can be used in listings and all kinds of places.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

Comment (xendk) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss]: Yeah, but there should really only be one h1, as the more fun alternatives really isn't supported. W3C has even amended their recommendation: https://github.com/w3c/html/issues/169

Sucks for us, but in light of that multiple h1's is really an error, I think it's reasonable to globally style it.

Comment (naxoc) [reply web/themes/custom/kk_basis/src_assets/scss/components/navigion_page.scss]: Alright. I'll give it a thumbs up. But there is no way to enforce this though.

**Pull request merged**